### DISCUSSION TITLE
Why does JavaPackager need  to use main.class=WinRun4JLauncher for WinRun4J?

### DISCUSSION THREAD

**Discussion by sproket** [fvarrui/JavaPackager#407] [Q&A] [answered]
I noticed this in the INI file generated.   I tried editing this file and putting in my actual main class file and the app ran fine.   How come you needed this?  

TIA

**Comment by fvarrui** [comment, marked as answer]
Yes, you are right, we don't need WinRun4JLauncher. But don't worry about that, as it's fixed in 1.7.6-SNAPSHOT.
Thanks!!

**Comment by fvarrui** [comment]
That was probably the result of trying to do my best late at night 😄
